scripts/updateConsent.js

```diff
@@ -18,14 +18,15 @@ function batchPromises(batchSize, emails, fn){
 
 function updateConsent(database){
   //Still need to grab emails from a sensitive data file
-  const emails = JSON.parse(fs.readFileSync('./tmp/consented-latest.json'))
+  // const emails = JSON.parse(fs.readFileSync('./tmp/consented-latest.json'))
+  const emails = JSON.parse(fs.readFileSync('./tmp/test.json'))
```

Comment:
This seems like the wrong filename :)

Reply:
Oh yes, this needs to be changed. Again, reopen this branch or just do a patch?

I also might need some help figuring out how to run this script on heroku 😭

Reply:
Hmmm. actually, it seems like I fixed this already in commit https://github.com/mit-teaching-systems-lab/threeflows/commit/6b1b2762302b8d6c601d6e83798009eda79e4744